snap/info.go

```diff
@@ -997,3 +997,28 @@ func DropNick(nick string) string {
 	}
 	return nick
 }
+
+// StoreName splits the instance name and returns the store name of the snap.
+func StoreName(name string) string {
+	return strings.SplitN(name, "_", 2)[0]
+}
+
+// SplitInstanceName splits the instance name and returns the store name and the
+// instance key.
+func SplitInstanceName(name string) (store string, instanceKey string) {
```

Comment:
Same request about `name -> instanceName`. You can probably abbreviate return tuple to `(store, instanceKey string)`

Reply:
renamed and tweaked the return list